app/layout/projects/new/form/planning-area-grid-uploader/component.tsx

```diff
@@ -95,7 +91,14 @@ export const PlanningAreaGridUploader: React.FC<PlanningAreaGridUploaderProps> =
 
   const onDropRejected = (rejectedFiles) => {
     const r = rejectedFiles[0];
-    const { errors } = r;
+
+    // `file-too-large` backend error message is not friendly.
```

Comment:
Not really a fan of repeating this bit of code in all uploaders, but given the customisation of the message (custom max size) I'm not sure how to go about it. It feels too specific to create an util for it, but could be done. What do you think?

Reply:
I think the best thing would be to have an upload component for everything, but I don't know how agnostic we would get to do it since the upload logic is not the same for all cases. 

We can look together if it is feasible or a priority to do so.